### ISSUE TITLE
System Status Check Page still broken

### ISSUE THREAD

**Issue by Padrio** [mail-in-a-box/mailinabox#1182]
Seems like #1134 still haven't been fully resolved. When opening the page i still receive the '''Something went wrong, sorry.''' Message.

When running the check-script from command line i receive the following output:

```
root@mail ~/mailinabox/management # ./status_checks.py 

System
======
✓  All system services are running.
✖  The SSH server on this machine permits password-based login. A more secure way to log in is using a public key. Add 
   your SSH public key to $HOME/.ssh/authorized_keys, check that you can log in without a password, set the option 
   'PasswordAuthentication no' in /etc/ssh/sshd_config, and then restart the openssh via 'sudo service ssh restart'.
✓  System software is up to date.
✓  Mail-in-a-Box is up to date. You are running version v0.22.
✓  System administrator address exists as a mail alias. [administrator@mail.openmind-shop.de ↦ service@openmind-shop.de]
✓  The disk has 42.33 GB space remaining.
✓  System memory is 47% free.

Network
=======
✓  Firewall is active.
✓  Outbound mail (SMTP port 25) is not blocked.
✓  IP address is not blacklisted by zen.spamhaus.org.
multiprocessing.pool.RemoteTraceback: 
"""
Traceback (most recent call last):
  File "/usr/lib/python3.4/multiprocessing/pool.py", line 119, in worker
    result = (True, func(*args, **kwds))
  File "/usr/lib/python3.4/multiprocessing/pool.py", line 47, in starmapstar
    return list(itertools.starmap(args[0], args[1]))
  File "./status_checks.py", line 344, in run_domain_checks_on_domain
    check_primary_hostname_dns(domain, env, output, dns_domains, dns_zonefiles)
  File "./status_checks.py", line 396, in check_primary_hostname_dns
    if ip == env['PUBLIC_IP'] and not (ipv6 and env['PUBLIC_IPV6'] and normalize_ip(ipv6) != normalize_ip(env['PUBLIC_IPV6'])):
  File "./status_checks.py", line 897, in normalize_ip
    return str(ipaddress.ip_address(ip))
  File "/usr/lib/python3.4/ipaddress.py", line 54, in ip_address
    address)
ValueError: '[Not Set]' does not appear to be an IPv4 or IPv6 address
"""

The above exception was the direct cause of the following exception:

Traceback (most recent call last):
  File "./status_checks.py", line 980, in <module>
    run_checks(False, env, ConsoleOutput(), pool)
  File "./status_checks.py", line 65, in run_checks
    run_domain_checks(rounded_values, env, output, pool)
  File "./status_checks.py", line 323, in run_domain_checks
    ret = pool.starmap(run_domain_checks_on_domain, args, chunksize=1)
  File "/usr/lib/python3.4/multiprocessing/pool.py", line 268, in starmap
    return self._map_async(func, iterable, starmapstar, chunksize).get()
  File "/usr/lib/python3.4/multiprocessing/pool.py", line 599, in get
    raise self._value
ValueError: '[Not Set]' does not appear to be an IPv4 or IPv6 address
root@mail ~/mailinabox/management # `

Im running the latest Version of mail-in-a-box with Python Version 2.7.6 and a external DNS Setup.
If you need any more informations just tell me

**Comment by yodax** [comment]
This isn't released yet. Are you running master?

**Comment by Padrio** [comment]
I've installed by following the usual instructions `curl -s https://mailinabox.email/setup.sh | sudo bash`
Is there a way to checkout the current master-state?

**Comment by Padrio** [comment]
Update: Just updated to master, fix from #1134 is working.
